src/utils.js

```diff
@@ -385,6 +385,22 @@ export function basicCompatVersionComparison(versionAdded, minVersion) {
   return !Number.isNaN(asNumber) && asNumber > minVersion;
 }
 
+function firstStableVersion(supportInfo) {
+  return supportInfo.reduce((versionAdded, supportEntry) => {
+    if (
+      !Object.prototype.hasOwnProperty.call(supportEntry, 'flags') &&
```

Comment:
Should we consider the API still locked behind a flag if there is a flags property but it is an empty array?
I guess that we may not have any actual entry in the `mdn/browser-compat-data` repo that does have an empty array as its `flags` property, but I guess that it may be something still possible to happen in the future (and likely not be noticed until it does trigger a visible validation error to an extension developer).

Reply:
That's what https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data.schema.json#L20 is for :)

Reply:
ah, that's good, I didn't know exactly what the browser-compat-data was linting/testing explicitly and so I wanted to mention it.

I forgot to mention in my last round of comments, but I was thinking that it would be nice if this kind of helpers would be provided by the browser-compat-data npm package itself, because the automation of that github project could ensure that the helper is in sync with any schema change may happen in a future version.

Not a blocking issue, but I wanted to ask you what is your opinion about that, especially given your first hand experience on using the browser-compat-data package as an addons-linter dependency.

Reply:
I think that would if anything be a separate package probably, though currently it seems like most users of BCD are rolling their own logic. And most people seem to be using slightly different logic and filters they apply to the data.
Further, these helpers here are very simplistic and ignore a bunch of things that BCD _could_ describe.